Makefile

```diff
@@ -51,11 +51,33 @@ fix-all:
 	make black-fix
 	make lint-fix
 
+# Download binaries from Taskcluster to run tests
+# See https://firefox-ci-tc.services.mozilla.com/tasks/index/translations.cache.level-1.toolchains.v3.fast-align/latest
```

Comment:
Suggestion: `See https://firefox-ci-tc.services...` This would be better documented in the binary file.

Reply:
moved